topics/templates/document-attributes-product.adoc

```diff
@@ -113,14 +113,14 @@ endif::[]
 :jdgserver_version: 7.3.0
 :jdgserver_crossdcdocs_link: https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html/red_hat_data_grid_user_guide/x_site_replication
 
-:fuseVersion: JBoss Fuse 6.3.0 Rollup 10
+:fuseVersion: JBoss Fuse 6.3.0 Rollup 11
```

Comment:
Updating Fuse versions should really be a separate PR

Reply:
The above PR would need to address the broken link test for product APIs. Guess simplest is just to ignore the link.

Reply:
> Updating Fuse versions should really be a separate PR

This is the version WF 17 upgrade has been tested with, so I'd say it belongs to here.

Reply:
> This is the version WF 17 upgrade has been tested with, so I'd say it belongs to here.

+1